lib/dai-plugin-mcd/src/index.js

```diff
@@ -71,29 +76,39 @@ const defaultCdpTypes = [
 ];
 
 export default {
-  addConfig: (_, { cdpTypes = defaultCdpTypes, addressOverrides } = {}) => {
+  addConfig: (
+    _,
+    { cdpTypes = defaultCdpTypes, network = 'mainnet', addressOverrides } = {}
```

Comment:
let's have `network` default to "testnet" for now so that the plugin's behavior stays the same